main.py

```diff
@@ -2,7 +2,7 @@
 import sys, os
 
 # Need to import all resources
-from api.cve import *
+from api.pcve import *
```

Comment:
I understand you need everything but convention still says you'll have to import all individually, `*` import creates the big problem that you don't know which imported elements conflict objects in this namespace when you go to define them.